src/main/resources/config/liquibase/changelog/20192405155100_add_metadata.xml

```diff
@@ -0,0 +1,41 @@
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="textType" value="varchar" dbms="postgresql, h2"/>
+    <property name="textType" value="text" dbms="oracle"/>
+
+    <changeSet id="20192405155100" author="ssenko">
+        <addColumn tableName="pocket">
+            <column name="metadata_value" type="${textType}" defaultValue="{}">
+            </column>
+        </addColumn>
+
+        <sql>UPDATE pocket SET metadata_value = '{}'</sql>
```

Comment:
can we avoid pre filling DB with such default value? I think it is more flexible to leave it empty and handle null value on back-end

Reply:
it's difficult to catch all side effects
because metadata it's identifier as label